release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -667,6 +667,146 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
+* *Changed*
+** Renamed `AggregatedAPIDown` to `KubeAggregatedAPIDown`.
+** Renamed `AggregatedAPIErrors` to `KubeAggregatedAPIErrors`.
+** Removed the `HighlyAvailableWorkloadIncorrectlySpread` alert. This alert is no longer needed because the Cluster Monitoring Operator ensures hard anti-affinity for all monitoring components.
+** Improved the description of the `KubeMemoryOvercommit` alert.
+** Excluded `ReadOnlyMany` volumes from the `KubePersistentVolumeFillingUp` alerts.
+** Improved `NodeFilesystemSpaceFillingUp` alerts to make it consistent with the Kubernetes garbage collection thresholds.
+** Extended `PrometheusOperator` alerts to include the Prometheus operator running in the `openshift-user-workload-monitoring` namespace.
+** Replaced the `ThanosSidecarPrometheusDown` and `ThanosSidecarUnhealthy` alerts by `ThanosSidecarNoConnectionToStartedPrometheus`.
+** Changed the severity of `KubeletTooManyPods` from `warning` to `info`.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for recording rules or alerting rules.
+=====
+
+[id="ocp-4-10-monitoring-changes-to-metrics"]
+==== Changes to metrics
+
+* Pod-centric cAdvisor metrics available at the slice level have been dropped.
+* The following metrics are now exposed:
+** `kube_poddisruptionbudget_labels`
+** `kube_persistentvolumeclaim_labels`
+** `kube_persistentvolume_labels metric`
+* Metrics with the name `kube_*annotation` have been removed from `Kube-state-metrics`.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for metrics.
+=====
+
+[id="ocp-4-10-monitoring-hard-anti-affinity-rules"]
+==== Added hard anti-affinity rules and pod disruption budgets for certain components
+
+With this release, hard anti-affinity rules and pod disruption budgets have been enabled for the following monitoring components to reduce down time during patch upgrades:
+
+* Alertmanager
++
+[NOTE]
+=====
+As part of this change, the number of Alertmanager replicas has been reduced from three to two.
+=====
++
+* Prometheus adapter
+* Prometheus
+* Thanos Querier
+
+If you have enabled user-defined monitoring, the following components also use these rules and budgets:
+
+* Prometheus
+* Thanos Ruler
+
+[id="ocp-4-10-monitoring-alert-routing-user-defined-project-tech-preview"]
+==== Alert routing for user-defined projects (Technology preview)
+
+This release introduces a Technology Preview feature in which an administrator can enable alert routing for user-defined monitoring. Users can then add and configure alert routing for their user-defined projects.
+
+
+[id="ocp-4-10-monitoring-alertmanager"]
+==== Alertmanager
+
+Access to the third-party Alertmanager web user interface from the {product-title} route has been removed.
+
+[id="ocp-4-10-monitoring-prometheus"]
+==== Prometheus
+
+* The Prometheus adapter now uses the Thanos Querier API instead of the Prometheus API.
+* {product-title} cluster administrators can now configure audit logs for the Prometheus adapter.
```

Comment:
`The Prometheus adapter now uses the Thanos Querier API instead of the Prometheus API.`
and
`{product-title} cluster administrators can now configure audit logs for the Prometheus adapter.`
they are for prometheus adapter, should be placed under new section `Prometheus Adapter`, not under `Prometheus`

Reply:
Created a new `Prometheus adapter` section and moved the items.